decentraland/unity-explorer
# fix: hide & show all debug widgets through /debug command

Hide & show the whole debug panel when typing `/debug` in the chat.

# Test steps

1. Start the app and check that the debug panel is hidden
2. Enter the world
3. Type `/debug` in the chat
4. Observe that the debug panel is fully visible with all the widgets
5. Type `/debug` in the chat again
6. Observe that the debug panel is fully hidden
7. Type `/debug help` or `/debug CHAT`
8. Observe that the correct widget is displayed
